tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -70,7 +70,11 @@ class CloudGov(Common):
     #
     env_based_db_name = f'tdp_db_{cloudgov_space_suffix}_{cloudgov_name}'
 
-    db_name = database_creds['db_name'] if (cloudgov_space_suffix in ["prod",  "staging"]) else env_based_db_name
+    logger.debug("css: " + cloudgov_space_suffix)
+    if (cloudgov_space_suffix in ["prod", "staging"]):
+        db_name = database_creds['db_name']
+    else:
+        db_name = env_based_db_name
```

Comment:
Isn't this change doing the same thing?

Reply:
The prior one-liner was hitting a KeyError off `db_name` despite the fact it should have been triggering the else clause so combining this with the other CGNAME update to hopefully avoid that w/o a try/except block.